orchagent/portsorch.cpp

```diff
@@ -3116,14 +3220,22 @@ bool PortsOrch::removeVlanMember(Port &vlan, Port &port)
     /* Restore to default pvid if this port joined this VLAN in untagged mode previously */
     if (sai_tagging_mode == SAI_VLAN_TAGGING_MODE_UNTAGGED)
     {
-        if (!setPortPvid(port, DEFAULT_PORT_VLAN_ID))
+        if (!setPortPvid(port, 0))
         {
             return false;
         }
     }
 
     m_portList[port.m_alias] = port;
     vlan.m_members.erase(port.m_alias);
+    if (port.m_oper_status == SAI_PORT_OPER_STATUS_UP)
```

Comment:
port.m_oper_status == SAI_PORT_OPER_STATUS_DOWN ?

Reply:
A port that is oper UP is being removed from vlan, so we decrement the up_member_cnt.